projects/giflib/Dockerfile

```diff
@@ -16,9 +16,10 @@
 
 FROM gcr.io/oss-fuzz-base/base-builder
 MAINTAINER esr@thyrsus.com
-RUN apt-get update && apt-get install -y make autoconf automake libtool wget zlib1g-dev
-RUN apt-get update && \
-apt-get install -y  libtool ninja-build cmake
+RUN apt-get update -y && \
+  apt-get install -y make autoconf automake libtool wget zlib1g-dev \
+  protobuf-compiler libprotobuf-dev binutils cmake \
```

Comment:
protobuf-compiler libprotobuf-dev
Are these needed since
-DLIB_PROTO_MUTATOR_DOWNLOAD_PROTOBUF=ON is set, can you please test.

Reply:
Just tested. Not needed. Great suggestion, thank you.